android/build.gradle

```diff
@@ -2,8 +2,8 @@ buildscript {
     ext.kotlin_version = '1.6.10'
     ext {
          minSdkVersion       = 21
-         compileSdkVersion   = 31
```

Comment:
It’s a bit soon to upgrade to sdk 33 I think, as android 13 is still in beta

Reply:
I believe one of the upgraded packages asked for this update. Either geolocator or flutter_map.

Reply:
Android 13 is out, so I guess we can upgrade again and merge this.

Might need to add notification request in a separate PR too